### PR TITLE
[BULK template] Add Transformer Dofn

### PR DESCRIPTION
**Pull request by Deep1998**
This PR adds the DoFN for transformer, that takes in a SourceRow generated by the reader and outputs a Spanner Mutation.

### CODE REVIEW

File: v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/transformer/SourceRowToMutationDoFn.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    } catch (Exception e) {
+      // TODO: Add DLQ integration once supported.
+      LOG.error(
+          "Unable to transform source row to spanner mutation: {} {}",
+          e.getMessage(),
+          e.fillInStackTrace());
+    }
+  }
+
+  static Mutation mutationFromMap(String spannerTableName, Map<String, Value> values) {
```
Comment (manitgupta) [inline v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/transformer/SourceRowToMutationDoFn.java:80]: I've seen this in other PRs as well - Is there a reason you do not add `private/public` access modifiers for methods? I think it's almost a convention to do that, right? 
For e.g `mutationFromMap` should be private IMO?

**Review by manitgupta — COMMENTED** [review]
Patch coverage is at 73% (Target: 80%)

File: v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/transformer/SourceRowToMutationDoFn.java (hunk 1 above)
Comment (Deep1998) [reply v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/transformer/SourceRowToMutationDoFn.java:80]: Yes I always thought not specifying defaults to private, just realised it is a separate access specifier. Will add :P

Comment (manitgupta) [reply v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/transformer/SourceRowToMutationDoFn.java:80]: Hehe, thanks. Please check other places as well once!

File: v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/transformer/SourceRowToMutationDoFn.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+      ISchemaMapper iSchemaMapper, Map<String, SourceTableReference> tableIdMapper) {
+    return new AutoValue_SourceRowToMutationDoFn(iSchemaMapper, tableIdMapper);
+  }
+
+  @ProcessElement
+  public void processElement(ProcessContext c) {
+    SourceRow sourceRow = c.element();
+    GenericRecord record = sourceRow.getPayload();
+    if (!tableIdMapper().containsKey(sourceRow.tableSchemaUUID())) {
+      LOG.error(
```
[the remark this answers is not in the document]
Comment (Deep1998) [reply v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/transformer/SourceRowToMutationDoFn.java:56]: Yes its not good practice, added this a a temporary measure. The next task is to work on migration metrics, which is when relevant counters etc will be added across the template, and errors will be handled via DLQ.

**Review by manitgupta — APPROVED** [review]

**Pull request merged**